taxonomy-editor-frontend/src/pages/editentry/LanguageSelectionDialog.tsx

```diff
@@ -40,29 +43,27 @@ const LanguageSelectionDialog = ({
             id="multiple-lang-checkbox"
             multiple
             value={newShownLanguageCodes}
-            onChange={handleLanguageCheckToggle}
+            onChange={(event) =>
+              setNewShownLanguageCodes(event.target.value as string[])
+            }
             input={<OutlinedInput label="Languages" />}
             renderValue={(selected) =>
               selected.map((langCode) => ISO6391.getName(langCode)).join(", ")
             }
           >
             {ISO6391.getAllNames()
               .sort()
-              .map(
-                (langName) =>
-                  langName !== mainLanguage.main_language && (
-                    <MenuItem key={langName} value={ISO6391.getCode(langName)}>
-                      <Checkbox
-                        checked={
-                          newShownLanguageCodes.indexOf(
-                            ISO6391.getCode(langName)
-                          ) > -1
-                        }
-                      />
-                      <ListItemText primary={langName} />
-                    </MenuItem>
-                  )
-              )}
+              .map((languageNameItem) => {
+                const languageCodeItem = ISO6391.getCode(languageNameItem);
+                return languageCodeItem === mainLanguageCode ? null : (
+                  <MenuItem key={languageCodeItem} value={languageCodeItem}>
+                    <Checkbox
+                      checked={newShownLanguageCodes.includes(languageCodeItem)}
+                    />
+                    <ListItemText primary={languageNameItem} />
+                  </MenuItem>
+                );
+              })}
```

Comment:
I think this issue was already fixed. Do you want to change this again?

Reply:
yep, there was a small issue with the main language